docs/user/openstack/deploy_sriov_workers.md

```diff
@@ -0,0 +1,378 @@
+            [Install]
+            WantedBy=local-fs.target
+```
+
+### Enabling the No-IOMMU feature for the RHOSP VFIO driver
+
+You can apply a machine config to your machine pool that enables the No-IOMMU feature for the Red Hat OpenStack Platform (RHOSP) virtual function I/O (VFIO) driver. The RHOSP vfio-pci driver requires this feature.
+
+```yaml
+kind: MachineConfig
+apiVersion: machineconfiguration.openshift.io/v1
+metadata:
+  name: 99-vfio-noiommu
+  labels:
+    machineconfiguration.openshift.io/role: worker
+spec:
+  osImageURL: ''
+  config:
+    ignition:
+      version: 2.2.0
+    storage:
+      files:
+      - filesystem: root
+        path: "/etc/modprobe.d/vfio-noiommu.conf"
+        contents:
+          source: data:text/plain;charset=utf-8;base64,b3B0aW9ucyB2ZmlvIGVuYWJsZV91bnNhZmVfbm9pb21tdV9tb2RlPTEK
+          verification: {}
+        mode: 0644
+```
+
+If you need to configure your deployment for real-time running or low latency, see the [performance addon operator documentation](performance-addon-operator), or you plan to use a technology such as DPDK that requires the use of [hugepages](huge-pages-perf-addon).
+
+Once your OpenShift control plane is running, you will need to install the SR-IOV Network Operator. To do so, you will need access to an account on your OpenShift cluster with `cluster-admin` privileges. Once you are logged into that account, follow [these steps](sriov-operator) to install the Operator. Then, [configure your SR-IOV network device](configure-sriov-network-device).
+
+## Creating SR-IOV Networks for Worker Nodes
+
+The SR-IOV networks that are attached to worker nodes need to be created before creating the nodes. Please reference the example of how to create radio and uplink provider networks in OpenStack.
+
+```sh
+# Create Networks
+openstack network create radio --provider-physical-network radio --provider-network-type vlan --provider-segment 120
+openstack network create uplink --provider-physical-network uplink --provider-network-type vlan --external
+
+# Create Subnets
+openstack subnet create --network radio --subnet-range <radio_network_subnet_range> radio
+openstack subnet create --network uplink --subnet-range <uplink_network_subnet_range> uplink
+```
+
+## Creating SR-IOV Worker Nodes in IPI
+
+Creating worker nodes as a post-install operation in IPI is handled by the machine-api. To create a new set of worker nodes in the machine-api, you will need to [create a new machineSet in OpenShift](openstack-machine-sets).
+
+```sh
+oc get machineset -n openshift-machine-api <machineset_name> -o yaml > sriov_machineset.yaml
+```
+
+When editing an existing machineSet, or a copy of one, to create SR-IOV worker nodes, add each subnet configured for SR-IOV to the machineSet's `providerSpec`. The example below will attach ports from the `radio` and `uplink` subnets, from the previous example, to all the worker nodes in the machineSet. For all SR-IOV ports, always set:
```

Comment:
This section needs to be reworded with the new method under the "port"  section of providerSpec